api/app/controllers/spree/api/v2/storefront/account/addresses_controller.rb

```diff
@@ -0,0 +1,71 @@
+module Spree
+  module Api
+    module V2
+      module Storefront
+        module Account
+          class AddressesController < ::Spree::Api::V2::BaseController
+            before_action :require_spree_current_user
+
+            def index
+              render_serialized_payload { serialize_collection(collection) }
+            end
+
+            def create
+              result = Spree::Account::Addresses::Create.call(user: spree_current_user, address_params: address_params)
```

Comment:
@kgorazd please use dependencies for this eg. https://github.com/spree/spree/blob/master/api/app/controllers/spree/api/v2/storefront/cart_controller.rb#L20

Reply:
fixed